app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true
+  validates :text, presence: true
```

Comment:
The Post model must have a method that returns the 5 most recent comments for a given post as it's part of project requirement. Kindly implement that 👍